src/platform/K32W/BLEManagerImpl.cpp

```diff
@@ -37,7 +58,1415 @@ namespace Internal {
 
 namespace {
 
+/*******************************************************************************
+ * Macros & Constants definitions
+ *******************************************************************************/
+/* Timeout of BLE commands */
+#define CHIP_BLE_KW_EVNT_TIMEOUT 300
+
+/** BLE advertisment state changed */
+#define CHIP_BLE_KW_EVNT_ADV_CHANGED 0x0001
+/** BLE advertisement command failed */
+#define CHIP_BLE_KW_EVNT_ADV_FAILED 0x0002
+/** BLE advertisement setup failed */
+#define CHIP_BLE_KW_EVNT_ADV_SETUP_FAILED 0x0004
+/** BLE advertisement parameters setup complete */
+#define CHIP_BLE_KW_EVNT_ADV_PAR_SETUP_COMPLETE 0x0008
+/** BLE advertisement data setup complete */
+#define CHIP_BLE_KW_EVNT_ADV_DAT_SETUP_COMPLETE 0x0010
+/** BLE random address set */
+#define CHIP_BLE_KW_EVNT_RND_ADDR_SET 0x0020
+/** BLE Initialization complete */
+#define CHIP_BLE_KW_EVNT_INIT_COMPLETE 0x0040
+/** BLE Received a handle value confirmation from the client */
+#define CHIP_BLE_KW_EVNT_INDICATION_CONFIRMED 0x0080
+/** BLE send indication failed */
+#define CHIP_BLE_KW_EVNT_INDICATION_FAILED 0x0100
+/** Maximal time of connection without activity */
+#define CHIP_BLE_KW_CONN_TIMEOUT 60000
+
+#define LOOP_EV_BLE (0x08)
+
+/* controller task configuration */
+#define CONTROLLER_TASK_PRIORITY (6U)
+#define CONTROLLER_TASK_STACK_SIZE (gControllerTaskStackSize_c / sizeof(StackType_t))
+
+/* host task configuration */
+#define HOST_TASK_PRIORITY (3U)
+#define HOST_TASK_STACK_SIZE (gHost_TaskStackSize_c / sizeof(StackType_t))
+
+/* ble app task configuration */
+#define CHIP_DEVICE_CONFIG_BLE_APP_TASK_PRIORITY (HOST_TASK_PRIORITY - 1)
+#define CHIP_DEVICE_CONFIG_BLE_APP_TASK_STACK_SIZE (1024)
+
+/* advertising configuration */
+#define BLEKW_ADV_MAX_NO (2)
+#define BLEKW_SCAN_RSP_MAX_NO (2)
+#define BLEKW_MAX_ADV_DATA_LEN (31)
+#define CHIP_ADV_SHORT_UUID_LEN (2)
+
+/* Message list used to synchronize asynchronous messages from the KW BLE tasks */
+static anchor_t blekw_msg_list;
```

Comment:
nit: keyword 'static' is not required in anonymous namespace

Reply:
Thanks, removed the 'static' occurrence in anonymous namespace.